patches/tools-perf-process_perf_results.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/perf/process_perf_results.py b/tools/perf/process_perf_results.py
+index 5009ca8a6306a1f30642f4bf3ecbdd9304ac0790..a2217cebbcc15f372b861134cdc095b5ddabc9e9 100755
+--- a/tools/perf/process_perf_results.py
++++ b/tools/perf/process_perf_results.py
+@@ -41,7 +41,7 @@ except ImportError:
+   pass
+ 
+ 
+-RESULTS_URL = 'https://chromeperf.appspot.com'
++RESULTS_URL = 'https://brave-perf-dashboard.appspot.com'; sys.path.append(os.path.join(path_util.GetChromiumSrcDir(), 'brave', 'tools', 'perf', 'components')); import fake_logdog as logdog_helper
```

Comment:
can't we add this in python paths in config.js?

Reply:
We can in theory.
I'm afraid of possible name conflicts with chromium local imports in other places.
Also it doesn't reduce the number of patched lines.

Reply:
since we're using `as` here anyway, couldn't we avoid that by prefixing the original with `brave_`? I'm not so much worried about reducing patches here, it's just easier when everything is grouped together and added in the same way

Reply:
Didn't catch you idea. Do you mean change `fake_logdog` to `brave_logdog_helper` ?

Reply:
yes, then there's no chance of any naming conflict

Reply:
ok, I will change this. Anything else? ;)